src/webview/git-import/gitImportLoader.ts

```diff
@@ -19,7 +19,7 @@ import { selectWorkspaceFolder } from '../../util/workspace';
 import { vsCommand } from '../../vscommand';
 import { loadWebviewHtml } from '../common-ext/utils';
 import { DevfileConverter } from './devfileConverter';
-import GitUrlParse = require('git-url-parse');
+import { gitUrlParse } from './gitParse';
```

Comment:
If you really want to include the `git-url-parse` source code in our repo, we should remove it as a dependency from the `package.json`, and instead add its dependencies to our `package.json`. This will help us avoid including two copies of the file (one under `src` and one under `node_modules`)

Reply:
Removed `git-url-parse`. But added `git-up` as dependency